node/sc/bridge_accounts.go

```diff
@@ -70,8 +81,32 @@ func (ba *BridgeAccounts) GetBridgeOperators() map[string]interface{} {
 	return res
 }
 
+// GetParentOperatorFeePayer can return the fee payer of parent operator.
+func (ba *BridgeAccounts) GetParentOperatorFeePayer() common.Address {
+	return ba.pAccount.feePayer
+}
+
+// SetParentOperatorFeePayer can set the fee payer of parent operator.
+func (ba *BridgeAccounts) SetParentOperatorFeePayer(feePayer common.Address) error {
+	ba.pAccount.feePayer = feePayer
+	ba.db.WriteParentOperatorFeePayer(feePayer)
+	return nil
+}
+
+// GetChildOperatorFeePayer can return the fee payer of child operator.
+func (ba *BridgeAccounts) GetChildOperatorFeePayer() common.Address {
+	return ba.pAccount.feePayer
```

Comment:
```suggestion
	return ba.cAccount.feePayer
```

Does 'cAccount' means child chain accounts?

Reply:
Good point. Thanks. :)